src/app/component/CheckoutBilling/CheckoutBilling.component.js

```diff
@@ -51,10 +60,47 @@ class CheckoutBilling extends PureComponent {
         this.setState({ isOrderButtonEnabled });
     };
 
-    setOrderButtonVisibility = (isOrderButtonVisible) => {
-        this.setState({ isOrderButtonVisible });
+    setTermsAndConfitionsAccepted = () => {
+        this.setState(({ termsAndConditionsAccepted: oldTermsAndConfitionsAccepted }) => ({
+            termsAndConditionsAccepted: !oldTermsAndConfitionsAccepted,
+            isOrderButtonEnabled: !oldTermsAndConfitionsAccepted
+        }));
     };
 
+    handleShowPopup = (e) => {
+        const { showPopup } = this.props;
+        e.preventDefault();
+        showPopup();
+    };
+
+    renderTermsAndConditions() {
+        const { termsAreEnabled } = this.props;
+        const { termsAndConditionsAccepted } = this.state;
+
+        if (!termsAreEnabled) return null;
+
+        return (
+            <>
+                <button
+                  block="CheckoutBilling"
+                  elem="Link"
+                  onClick={ this.handleShowPopup }
+                >
+                    { __('I agree to terms and conditions.') }
+                </button>
```

Comment:
Please do use button. Prefer label, this is not intuitive. Do it like:

I agree to [terms and conditions](http://openpopup.com) ☑️

Then on click on "terms and conditions" link - popup opens.